hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpcengine/net/AsyncSocket.java

```diff
@@ -212,6 +212,10 @@ protected void close0() throws IOException {
 
     @Override
     public final String toString() {
-        return getClass().getSimpleName() + "[" + localAddress + "->" + remoteAddress + "]";
+        if (clientSide) {
+            return getClass().getSimpleName() + "[" + localAddress + "->" + remoteAddress + "]";
+        } else {
+            return "               " + getClass().getSimpleName() + "[" + localAddress + "->" + remoteAddress + "]";
```

Comment:
Can you flip the arrow? This way we can see if socket is an active (created client side) or passive (created server side).

Reply:
So for client this should be
`[clientAddress -> serverAddress]`
and for server
`               [serverAddress <- clientAddress]`?

Reply:
Yes. This way you can see if the socket is created as 'active' or 'passive'.